### ISSUE TITLE
Missing visual hint on error for tags field

### ISSUE THREAD

**Issue by quantavi** [TrestleAdmin/trestle#280]
We are using following line for form to provide tags:
```ruby
col(xs: 6) {select :tag_ids, Tag::all.alphabetical, {label: "Tags"}, multiple: true}
```
And in model we check for the presence of tags:
```ruby
class Company < ApplicationRecord 
  has_and_belongs_to_many :tags, -> { alphabetical }
  validates_presence_of :tags
end
```
We correctly receive alert "tags: can't be blank", but no visual hint around tags field. Are we missing some code?

<img width="1407" alt="Screenshot 2020-02-13 at 14 08 59" src="https://user-images.githubusercontent.com/32466875/74438695-06286800-4e6b-11ea-9d42-6fe5c07a9487.png">

**Comment by quantavi** [comment]
I was able to create a workaround for this by adding custom validator to model:
```ruby
class Company < ApplicationRecord
  has_and_belongs_to_many :tags, -> { alphabetical }
  validate :any_tag

  def any_tag
    if self.tags && self.tags.length == 0
      errors.add(:tag_ids, I18n.t('validators.tags'))
    end
  end
end
```

**Comment by spohlenz** [comment]
Thank you for the report. This should now be fixed in c6ca5a7.
